src/controller/python/test/test_scripts/network_commissioning.py

```diff
@@ -119,6 +119,12 @@ async def test_wifi(self, endpointId):
         if res.networkingStatus != Clusters.NetworkCommissioning.Enums.NetworkCommissioningStatus.kSuccess:
             raise AssertionError(f"Unexpected result: {res.networkingStatus}")
 
+        # Arm the failsafe before making network config changes
```

Comment:
@Damian-Nordic where are we testing that the commands are rejected if the fail-safe is not armed?  Should add YAML tests for that...